src/domUpdates.js

```diff
@@ -62,10 +62,26 @@ const renderGrid = () => {
   recipeGrid.innerHTML = createGridHTML(gridData);
 }
 
+const showRecipe = (recipeCard) => {
+  const recipeCardName = recipeCard.lastElementChild.textContent;
+  const thisRecipe = recipeData.find(recipe => recipe.name === recipeCardName);
+  clickedRecipe.classList.toggle("hidden");
+  clickedRecipe.classList.toggle("flex");
+  clickedRecipe.classList.toggle("fade-in");
+};
+
+const closeRecipe = () => {
+  clickedRecipe.classList.add("hidden");
+  clickedRecipe.classList.remove("flex");
+  clickedRecipe.classList.remove("fade-in");
```

Comment:
Ayy SWEET!

Reply:
If we end up having to do a lot of this class toggling, I can make a helper that can clean it up a bunch too